src/sageworks/utils/extract_model_artifact.py

```diff
@@ -111,13 +175,13 @@ def download_and_extract_model(model_artifact_uri):
     """Exercise the ExtractModelArtifact class"""
 
     # Create the Class and test it out
-    my_endpoint = "abalone-regression-end"
+    my_endpoint = "solubility-with-shap-end"
     ema = ExtractModelArtifact(my_endpoint)
 
-    # Test the lower level methods
-    model_data_uri = ema.get_model_data_uri()
-    print(f"Model Data URI: {model_data_uri}")
-    my_model = ema.download_and_extract_model(model_data_uri)
+    # # Test the lower level methods
+    # model_data_uri = ema.get_model_data_uri()
+    # print(f"Model Data URI: {model_data_uri}")
+    # my_model = ema.download_and_extract_model(model_data_uri)
```

Comment:
Why are these commented out? I'll assume this wasn't intended for final and after PR merge, I'll uncomment these for testing purposes.